README.md

```diff
@@ -314,6 +315,7 @@ _Source:_ [What is Docker](https://www.docker.com/why-docker/)
 -   [docker-bench-security](https://github.com/docker/docker-bench-security) - script that checks for dozens of common best-practices around deploying Docker containers in production. By [@docker][docker]
 -   [docker-explorer](https://github.com/google/docker-explorer) - A tool to help forensicate offline docker acquisitions by [@Google][google]
 -   [docker-lock](https://github.com/safe-waters/docker-lock) - A cli-plugin for docker to automatically manage image digests by tracking them in a separate Lockfile. By [@safe-waters][safe-waters]
+-   [dvwassl](https://github.com/Peco602/dvwassl) - SSL-enabled Damn Vulnerable Web App to test Web Application Firewalls. By [Peco602](https://github.com/Peco602)
```

Comment:
@Peco602 is the difference between your version and https://hub.docker.com/r/vulnerables/web-dvwa the certificate?

Reply:
Yes. The standard DVWA is not able to work on HTTPS. My modification adds the TLS layer via default or custom certificate. I know it is not a big modification but it is very useful to perform Web Application Firewall rules validation in case the internal host to be protected must work on HTTPS.